pkg/analyze/ceph.go

```diff
@@ -204,6 +214,8 @@ func detailedCephMessage(msg string, status CephStatus) string {
 		msg = fmt.Sprintf("%s. OSD disk is full", msg)
 	} else if status.OsdMap.OsdMap.NearFull {
 		msg = fmt.Sprintf("%s. OSD disk is nearly full", msg)
+	} else if chkMsg, ok := status.Health.Checks["POOL_NO_REDUNDANCY"]; ok {
```

Comment:
How about instead of hardcoding this specific check, we iterate over all checks in health is not OK.